apps/form-builder/src/app/api/forms/route.ts

```diff
@@ -0,0 +1,22 @@
+import { createForm } from "@/server/actions/forms";
+import { zCreateFormRequest } from "@hack4impact-utk/internal-models";
+import { NextRequest, NextResponse } from "next/server";
+
+export async function POST (request: NextRequest) { 
+
+    
+    const requestBody = await request.json();
+    const variable = zCreateFormRequest.safeParse(requestBody);
+    if (variable.success) {
+        const form = await createForm(requestBody);
+        
+        
+        return NextResponse.json({_id:form._id}, {status: 201})
+    }
+    else {
+        return NextResponse.json({}, {status: 400})
```

Comment:
Should have mentioned this when we were discussing, my apologies. Could you include a message that says why we are returning a 400 (which indicates an error)? If the code reaches this `else`, it means there was an invalid request body (because it failed validation). [Here](https://github.com/hack4impact-utk/internal-applications/blob/main/apps/form-builder/src/app/api/forms/%5BformId%5D/formSubmissions/route.ts) is an existing code example where we give a custom message

Reply:
dunno